tools/cpplint.diff

```diff
@@ -1,123 +1,45 @@
 This diff file tracks the modifications that have been applied to cpplint.py compared to the original found here:
 https://github.com/cpplint/cpplint/blob/7b88b68187e3516540fab3caa900988d2179ed24/cpplint.py
```

Comment:
It seems like you have imported a new version, so the base reference should be updated?

Reply:
Seems like this got lost, will update...